email_app/app/logging.js

```diff
@@ -0,0 +1,25 @@
+import { app } from 'electron';
+const log = require( 'electron-log' );
+const pkg = require('../package.json');
+
+if(log.transports) {
+  // Log level
+  // error, warn, info, verbose, debug, silly
+  log.transports.console.level = 'verbose';
+  
+  /**
+  * Set output format template. Available variables:
+  * Main: {level}, {text}
+  * Date: {y},{m},{d},{h},{i},{s},{ms}
+  */
+  log.transports.console.format = '{level} {h}:{i}:{s}:{ms} {text}';
+  log.transports.file.file = `./${pkg.name}-nodejs.log`;
```

Comment:
I don't think the log filename should contain `-nodejs` since this are the logs generated by the app itself, the fact that the app is written in nodejs is implementation specific which should affect the log filename.

Also, I think it should be better to create the log file in the same path as where the underlying native lib creates it. You should be able to use the safe-app-nodejs `getLogPath` function to construct the path for you, i.e. ```this.props.getLogPath(`${pkg.name}.log`)```